.travis.yml

```diff
@@ -45,6 +45,7 @@ before_install:
   - rm /tmp/pandoc-2.2-1-amd64.deb
   - R -q -e "install.packages(c('countrycode','maps','readr','dplyr','tidyr','plotly','ggplot2','htmlwidgets','svglite'))"
   - Rscript -e 'sessionInfo()'
+  - python -m pip3 install numpy scipy matplotlib ipython jupyter pandas 
```

Comment:
at least locally, I do `pip3 install ... --user` (or `sudo pip3 install ...`). It looks like with the `python -m` syntax, here it should be `python3 -m pip install ...`

Reply:
see log here: https://travis-ci.com/carpentries/amy-feeds/builds/128949863#L6124